src/components/Courses/Courses.jsx

```diff
@@ -16,6 +18,28 @@ const Courses = () => {
 
   const navigate = useNavigate();
 
+  let courses = useSelector((state) => state.courses.courses);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:4000/courses/all')
+      .then(function (response) {
+        dispatch(addCourse(response.data.result[0]));
+      })
+      .catch(function (error) {
+        console.log(error.response);
+      });
+    axios
```

Comment:
You can combine 2 requests into handling process with help of [`axios.all`](https://blog.logrocket.com/using-axios-all-make-concurrent-requests/)